ms-oidc/src/main/java/net/trajano/ms/oidc/internal/IssuerConfig.java

```diff
@@ -137,6 +146,13 @@ public URI getUri() {
         return uri;
     }
 
+    public boolean isOpAllowed(final String op) {
+
+        // TODO implement this later.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)